src/components/accession2/withdraw/WithdrawModal.tsx

```diff
@@ -162,6 +178,32 @@ export default function WithdrawDialog(props: WithdrawDialogProps): JSX.Element
     onClose();
   };
 
+  const setIndividualError = (id: string, error?: string) => {
+    setFieldsErrors((prev) => ({
+      ...prev,
+      [id]: error,
+    }));
+  };
+
+  const validateAmount = () => {
+    if (record.withdrawnQuantity?.quantity) {
+      if (isNaN(record.withdrawnQuantity.quantity)) {
+        setIndividualError('withdrawnQuantity', strings.INVALID_VALUE);
+        return false;
+      } else {
+        if (Number(record.withdrawnQuantity.quantity) <= 0) {
+          setIndividualError('withdrawnQuantity', strings.INVALID_VALUE);
+          return false;
+        }
+      }
+    } else {
+      setIndividualError('withdrawnQuantity', strings.REQUIRED_FIELD);
```

Comment:
if quantity is 0 we will end up here? if so, maybe we need a different message for quantity=0?

Reply:
0 is '0' as string, so you end up in the invalid value message

Reply:
ah yes, forgot this is a string input